packages/block-library/src/query/edit/enhanced-pagination-modal.js

```diff
@@ -76,8 +76,7 @@ export default function EnhancedPaginationModal( {
 				<VStack alignment="right" spacing={ 5 }>
 					<span id={ modalDescriptionId }>{ notice }</span>
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
Hi @mirka @ciampo, I was not able to get this visually on the editor, tried some hacks as well, like to directly call this when adding the block, but did not get any success.

Can you help with this? All others screenshot has been attached.

Thank You,